homeassistant/components/minecraft_server/sensor.py

```diff
@@ -40,6 +50,7 @@ class MinecraftServerEntityDescriptionMixin:
 
     value_fn: Callable[[MinecraftServerData], StateType]
     attributes_fn: Callable[[MinecraftServerData], MutableMapping[str, Any]] | None
+    enabled_fn: Callable[[MinecraftServerType], bool]
```

Comment:
Why not just make it return a list:
```suggestion
    supported_server_types: list[MinecraftServerType]
```

Reply:
That sounds like a better idea. I'll try this later, thanks :+1: